web-client/src/modules/requests/components/RequestList.tsx

```diff
@@ -23,20 +24,18 @@ const RequestList: React.FC<RequestListProps> = ({
     if (requests) {
       const internalRequestList: React.ReactElement<any>[] = [];
       const internalRequestsRendered: Record<string, boolean> = {};
-
       for (const id in requests) {
         if (id && requests[id] && !requestsRendered[id]) {
           internalRequestsRendered[id] = true;
           internalRequestList.push(
             <RequestItem
               key={id}
-              request={requests[id].getRequest()}
-              handleRequest={(action?: boolean) =>
-                handleRequest && handleRequest(id, action)
-              }
+              // TODO: (es) remove requestId={id} // TODO: (es) Use key instead?  what is key used for?
+              request={requests[id]}
+              // TODO: (es) remove handleTimeline={handleTimeline}
               isCavAndOpenRequest={isCavAndOpenRequest}
               isPinAndOpenRequest={isPinAndOpenRequest}
```

Comment:
offers should be removed.  It is no up to RequestItem to kick off the query.  RequestItem should be split into RequestItemContainer and RequestItem.